int8 bumper
int16 left
int16 front_left
int16 center_left
int16 center_right
int16 front_right
int16 right
